src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  height: 406px;
+  width: 200px;
+  margin-right: 50px;
+
+  border-radius: 5px;
+  border: 1px solid #{$color-white};
```

Можна і без інтерполяції
```suggestion
  border: 1px solid $color-white;
```